native/src/main/java/io/ballerina/stdlib/xslt/XsltTransformer.java

```diff
@@ -79,11 +82,13 @@ public static Object transform(BXml xmlInput, BXml xslInput) {
             StringWriter stringWriter = new StringWriter();
             StreamResult streamResult = new StreamResult(stringWriter);
 
-            Transformer transformer = TransformerFactory.newInstance().newTransformer(xslSource);
+            Transformer transformer = new BasicTransformerFactory().newInstance().newTransformer(xslSource);
+            errStream.println(transformer);
```

Comment:
Is it necessary?

Reply:
Updated in [cfe19f6](https://github.com/ballerina-platform/module-ballerina-xslt/pull/427/commits/cfe19f6c1557bd6f2848b10770c41649fc72e896)